### DISCUSSION TITLE
Data preparation: How to download split files

### DISCUSSION THREAD

**Discussion by dainv1989** [Nicholasli1995/EgoNet#4] [Q&A]
Hi Nicholas,
I'm trying to reproduce inference demo. In data preparation, you wrote "Download the split files here" but I cannot see embedded link. Can you tell me where to download split files.
Thank you so much!

**Comment by Nicholasli1995** [comment]
> Hi Nicholas,
> I'm trying to reproduce inference demo. In data preparation, you wrote "Download the split files here" but I cannot see embedded link. Can you tell me where to download split files.
> Thank you so much!

Hi, I have updated the link and it should be available now.
